docs/site/examples.md

```diff
@@ -112,6 +112,24 @@ examples:
     links: 
     - language: powercli
       url: "/tree/master/examples/powercli/ha-restarted-vms"
+
+  - title: VMware Cloud on AWS SDDC Provisioned and Deletion Slack Notification
+    usecases:
+    - item: notification
+    id: vmware-cloud-ngw-slack
+    description: Send Slack message when a VMware Cloud on AWS SDDC is Provisioned or Deleted.
+    links:
+    - language: powershell
+      url: "/tree/master/examples/powershell/vmware-cloud-ngw-slack"
+
+  - title: VMware Cloud on AWS SDDC Provisioned and Deletion Microsoft Teams Notification
+    usecases:
+    - item: notification
+    id: vmware-cloud-ngw-teams
+    description: Send Microsoft Team message when a VMware Cloud on AWS SDDC is Provisioned or Deleted.
+    links:
+    - language: powershell
+      url: "/tree/master/examples/powershell/vmware-cloud-ngw-teams"
```

Comment:
So once this PR is in, this likely won't work due to the fn not being merged in master before the next release, correct?

Reply:
Yea ... I followed previous PR submitted by community on updating docs or we'll forget/lose context